broker-core/src/test/java/io/zeebe/broker/workflow/timer/TimerCatchEventTest.java

```diff
@@ -211,74 +245,102 @@ public void shouldCompleteTimerEvent() {
   @Test
   public void shouldTriggerTimerWithZeroDuration() {
     // given
+    final String processId = Strings.newRandomValidBpmnId();
+    final String timerId = Strings.newRandomValidBpmnId();
```

Comment:
No need to generate the element id if you filter the records by workflow instance key. Random id may make it harder to understand the records if the test fails.